### ISSUE TITLE
Fashion Street

### ISSUE THREAD

**Issue by snickerr** [decentraland/districts#31]
For the latest updates on our district plan please join us on Discord.
Decentraland: https://dcl.gg/discord (Look for fashion-street channel)
Fashion Street: https://discord.gg/xuDf6k7

**Comment by dclancy13** [comment]
That's a great idea! For consistency and to allow contributors to asses projects equally, could you include at the beginning of your proposal a filled in "District Proposal Framework"
Thanks so much for your contribution!

**District Proposal Framework**

Title:  **Name of Project**
Description: **Discuss in a paragraph the details of your Proposal**
Project Lead: **Name and Email**
Project Team :**Name / Email / Roles of each committed team member**
(list no more than 5 members)
 Goal:  **What does this District want to accomplish (i.e. generate revenue for its owners, act as a public gathering place, just be for fun) ** 
Ownership: **How will ownership of the district be distributed?**
LAND Needed: **How Much LAND will be required to successfully launch this District**
Startup Costs: **How much (in ETH) will be needed to fund building the District (pay for developers / designers / marketing)**
How will funds be raised: **Explain how startup costs will be raised to complete and launch the District**
How will contributors see a return value: **Explain why others would want to participate in this District**
Attachments / Designs / Budgets : Upload any supporting files.

Many District proposals may be too early on in the development process to answer all of these questions but the idea is to establish a common rubric for submitting a proposal, to get contributors to start thinking about how they intend to execute on their projects and to give potential supporters an idea of how far along in the development process each particular proposal is. This will help them make a well informed decision in choosing which project to participate in.

**Comment by saskiamueller** [comment]
Hi, I'm very interessted in your project. I have only some Mana Token until now, but I follow you. Let's have a Great Future in Decentraland City!

**Comment by snickerr** [comment]
Hi saskiamueller, glad you like the project. No need for too much mana here, hopefully we get some tiles around 0,0.

**Comment by Denis17DCL** [comment]
I like the idea
Put 30 land parcels a  week ago
Thinking of adding more, some time after the districts are being added now.

**Comment by snickerr** [comment]
Happy to hear that @Denis17DCL and welcome to Fashion Street!

**Comment by snickerr** [comment]
Fashion Street – Project Update 1

First of all we would like to thank you for your interest in our project and welcome you on Fashion Street. This is the first follow up which contains further details about the project.

Fashion Street will be a small but elegant district in DCL. From an urban/architectural perspective it will be similar to the Graben, which is Vienna’s main shopping street. Like the Graben (see picture below), it will have the form of a large, elongated square with different stores (clothing, accessories, art, jewelry, etc.) on both sides. 

![image](https://user-images.githubusercontent.com/32387344/31312600-8ca76488-abc7-11e7-8016-4aed15393c42.png)


Apart of shopping, the main attraction of Fashion Street will be the street itself. The street will be 30 meters  wide (approx 3 lands/parcels) exactly the same as the Graben and will be ornamented with temporary or permanent installations and sculptures from different installation artists. 

The stores 

The stores should be big and luxurious in order to give a great and high sensory experience to the clients. After the feedback received, we decided to increase the minimum size of a store and ornate all of them with astonishing walls and futuristic (if approved by the brand) furniture and accessories. After all, gold is cheap in DCL :)

All the stores will be created and customized after a consultation with both the land owners and the brands. Customization will involve (apart of the luxurious 3D interior design) single 3D scans for each item. Data such as item description, price, size etc. can be easily imported through APIs but 3D scanning has to be done manually. The total cost of the turnkey store will be determined on an individual basis, according to the brand promoted and the level of design/specific requests. We are in the early stages of a new era for shopping and very excited to be pioneers.

After the customization and integration with a blockchain based software for payment, accounting and invoicing, the store will be ready for VR shopping in DCL. 

A good practice would be to have a store manager in each store, who welcomes the clients and instructs them (especially the first-time VR clients) about the technology and how they can receive in a few days the same item they are virtually wearing. If land owners are busy or own many stores, they can hire a VR shopping assistant, which can become a great profession in the next few years. It can be done on a part-time basis, by people in different continents/time zones and at least initially, it would be like getting paid for playing a game.

There will be other follow-ups in the next few weeks with more updates. After the terraform event we can have a landowners discussion group in order to talk about future developments. Looking forward as well to organizing soon VR-meetings in DCL.

**Comment by Denis17DCL** [comment]
Hi Snickerr. 
Several ideas. 
1. Do you think ordinary shop box is ok? I do not see it as fun:(
2. What do you think about 

This kind of shops

I think we have to give the luxury clients something very special. 
It should be really new and great experience.  
I would love to buy things in VR with really great experience like for example in the huge palace in the middle of the golden hall - you see your most wanted Louis Vuitton clutch :)

What do you think?
Denis


Отправлено с iPhone

**Comment by Denis17DCL** [comment]
Another crazy idea is to have a vr mirror for clients. 
For that we need a hologram of the body of the client. 
So that the client could see her/himself with cloths on.
And post fotos to instagram from there:)
It can be a great idea for DCL - posting  your hologramfotos from VR to instagramm:) 
I do not know if it is technically possible now, but in future we should/must have it. 

Отправлено с iPhone

**Comment by snickerr** [comment]
@Denis17DCL Yes, that's the whole idea, having attractive and luxurious buildings.
And yes the "VR mirror" will be included in every store. Check the second image of the description ;)
As stated previously, in the next few weeks we are planning to have a meeting with landowners on a different channel in order to discuss about new ideas and future plans, so we can leave this space on github for major updates only.

**Comment by snickerr** [comment]
@Denis17DCL Thanks for the feedback! The drafts of the stores weren't that much luxurious and have been removed :)

**Comment by Denis17DCL** [comment]
I liked your first picture of the store which you had in github. 

Отправлено с iPhone

**Comment by Platosbuilder** [comment]
Hello,

What is the best method for communication with the project lead? Is there a chat channel or another way to organize? I have 5 land in this district and am excited to see what can be made. I was thinking of having the kind of cheaper store that sells btc tshirts and other gear. I would not mind having some land go to other general projects. Thanks for your time

**Comment by snickerr** [comment]
@Platosbuilder Thanks for reaching out, very happy to have you on-board. The best method of communication would be a forum with different sections, so discussions about the project, new ideas, technical questions, etc. can be conducted in a fast paced way. Hope to have it ready soon.

Regarding the store location within the district, the idea is to only allocate the luxury fashion brands in the main street. For all the other type of stores (small brands, sports brands, your btc t-shirt store etc.) we're thinking about a modern shopping center. 

By the way, why not selling DCL t-shirts in a Hard Rock Café style? "Hard Rock Café - DCL". That would be amazing but Hard Rock Café needs to approve it.

**Comment by Denis17DCL** [comment]
Great!!!
I like modern shopping Centre as well

Отправлено с iPhone

**Comment by Denis17DCL** [comment]
Btw, are we gonna limit amount of land needed for the project?
Or everyone is welcomed till December ?

Отправлено с iPhone

**Comment by snickerr** [comment]
Yes, after reaching the amount of land needed we will limit new land contributions.

**Comment by Denis17DCL** [comment]
How much is needed?

Отправлено с iPhone

**Comment by snickerr** [comment]
Basically land will cover the creation of a large shopping street and so far two big buildings (Shopping Center and Catwalk). The only size estimation we have now is the width of the main street which will be 7 parcels wide (3 parcels for the street and 2 parcels on each side for the stores/buildings). The length of the street and the size of the buildings are still to be determined. Will definitely send an update about this though.

**Comment by Denis17DCL** [comment]
Thanks. 

Отправлено с iPhone

**Comment by Denis17DCL** [comment]
we already have 1 kilometer long....with 700 lands

**Comment by Denis17DCL** [comment]
Please check rocket chat direct message

Отправлено с iPhone

**Comment by snickerr** [comment]
Fashion Street – Project Update 2

We’re getting closer to the terraform event and the DCL team are doing a great job by providing awesome tools which will make it easier for us to manage our district. 

We’re close to 1000 parcels which was our initial goal. The idea is to allocate around 400-500 parcels for:  1) The main shopping street  2) Some beautifully decorated squares in between the buildings  3) Some small adjacent streets (all converging in the main street) which give a feeling of a traditional shopping district.

The  two main buildings (Shopping Center and Catwalk) will be located at the two extremes of the main street and will have a size of around 300-400 parcels each. Obviously the more the better, especially for the Shopping Center which will be luxurious and multiple storey.

Our community of landowners is growing fast and after the terraform event all landowners will be invited to a private forum where we can discuss about new ideas and business development. In the meanwhile, if you have any question/suggestion or want to join the team as a Designer please PM me on chat.decentraland.org (username: snicker).

**Comment by ghost** [comment]
Hello,
I have a question regarding the feasibility of obtaining contracts within the fashion industry. Do you already have contacts that you are working with? If so, do they include the large brands mentioned in your proposal? I am looking to purchase parcels in your district, and this is the only concern I have regarding this project. As a newcomer, I am wondering if the Genesis City will be competition in the fashion industry, that is, will they offer shopping or will it be more business minded as I have been reading. Thanks in advance for your reply.

**Comment by snickerr** [comment]
@rodrgabr The large brands mentioned are just to give an example (don't have the contacts), since you find them in every shopping street. In my opinion, Genesis City will definitely have shopping stores directly competing with Fashion Street but that's the beauty of competition.

**Comment by ghost** [comment]
@snickerr Thanks for your quick reply. I definitely think there will be competition as well as you stated. However, with your vision of design and the fact that the entire district is devoted to the industry, we should see great brand names and companies take interest as the whole project takes off. I also like the fact to team up with Request to complete transactions. I'm still working on getting land staked in this great district. Do you plan on keeping the name of the district as it is now, or can that be changed at a later date?

**Comment by plato50** [comment]
Hi, How does it work to stake land to the main street, shopping mall and catwalk? How do we do that and when? Is there a map for it?

**Comment by snickerr** [comment]
Hi @plato50,  you need a minimum number of parcels in order to have a shop in the main street. For staking in the buildings you need just one land. Details are still to be determined though. As previously stated, we should be able to discuss details in our forum after the terraforming.

**Comment by Joeuro** [comment]
If I only opt in for one parcel of LAND during the event, can I add one later?  I want to be on the main street, but without knowing how many parcels I need, I'm kind of stuck.  Can you tell us how many will be required to be on the main street?

**Comment by snickerr** [comment]
@Joeuro That's still TBD, so can't give you a number.

**Comment by elpino** [comment]
Hi there i stake 1 parcel into this district

**Comment by Tonyrow** [comment]
Can i own a land in the district based on my contribution to run business? i believe that district is fine but idea is to have own property to rub business. I would like to have land based on contribution and run business as owner.

**Comment by bit802346** [comment]
which forum is under discussion or there is no forum yet?

**Comment by Tonyrow** [comment]
@bit802346 Fashion Market, i am a contributor and i would like to run own shop in the district for the product of fashion designers from my home country. I have developers to support project, so i would like to run a own shop there. let me know how the community is planning to mange this.

**Comment by Tonyrow** [comment]
Hi!

How are you?

Nice to meet you and thanks for reply. I would like to run some pilot project in Fashion Street District based on what my team is doing in VR.
I am participating in number of districts as well as auction for land in city, however it would be great to have run a VR place in district to test some model/avatar engines we have.
It would be great to get a land based on contribution. I am interested in opening a VR store to represent a fashion designers from my home country. I am ready to cover all costs including development and operations in VR and real world.

This will help district to have fast time to market means have some content ready for the visitors asap.

Let me know what are the Team plans for the development of district, any revenue models besides what you have on GitHub?
Mana contributors will have any option to own a land in the district?

My contribution address is 0x7e416744a422fe28da1a420323c5d0169A672401

I am ready to help with some ideas, reviews and other activities.

Thanks

________________________________

**Comment by ghost** [comment]
Hello, is there a forum open where we can join? I'm very excited to be a part of this project and am looking forward to hearing great ideas and seeing this become developed!

**Comment by snickerr** [comment]
Hey @Tonyrow, great to have you on-board! You're more than welcome to develop your ideas and run your own shop on Fashion Street. 
The on-boarding of contributors will start when the auction is over. Our forum is ready but you can access it by invitation only. Will publish soon a form to be filled by contributors, so they can receive the invitation code by email. PM me on DCL chat if you have any other question. Cheers

**Comment by bit802346** [comment]
HI! i'm waiting for invintation! it's great to start to develop fashion
street and make history!!!!

2017-12-20 23:36 GMT+03:00 snickerr <notifications@github.com>:

**Comment by archtad** [comment]
@snickerr Hello,
Architect looking to get involved in the aesthetic.
Worked on high end stores in NYC including Helmut Lang, Maserati Manhattan, J-Brand, Theory, etc.. Also owner of a 3D Modeling & Rendering company. Id like to get involved.

I have a few properties on Fashion Street currently in bid.

Please message me

**Comment by snickerr** [comment]
@bit802346 We'll probably start on-boarding of contributors in our forum in the first week of January. Will post here the link. 
@archtad Sounds good, please send me a PM on chat.decentraland.org. Cheers

**Comment by PHBear** [comment]
@snickerr When will the forum be launched?

**Comment by PHBear** [comment]
@snickerr And how will you ensure that only people that have commited to the project will have access to the forum?

**Comment by snickerr** [comment]
Fashion Street - Project update 3

After listening to the contributors, we decided to give them ownership of part of the land contributed. We agree that owning some land in our district is probably the best investment right now before VR shopping takes off in a couple of years.

Part of the land contributed will be used by the district to build internal roads and plazas which will further increase the value of land. We will try to give land adjacent to a road to as much contributors as possible. 

This means that there will not be any community owned buildings. In order to make profits from your store you need to work for it, have a dropshipping/affiliate contract with the brands and invest in marketing. It will work exactly like a Shopify store, but in in a decentralized VR world.

Everything else remains the same as stated in the initial description. If you want to have a turnkey store, our team will be happy to build it for you according to your needs. We have now different low poly store models to choose from. 
If you have development skills, you can choose a basic version of a store (think of it as a Shopify template) and decorate it with your own products or scan/create new items.

Fashion Street will maintain the decentralized spirit of DCL. So, if you can build stuff, you’re more than welcome to do that but it has to be in harmony with the district's theme. Unless your building is completely unrelated to the surrounding buildings, it will never be disapproved. If it’s unrelated to the district, we will submit it for community voting and it will be up to the community to make a decision. Remember that we aim to be “the elegant” district, so it’s probably going to be hard to pass the community test.

@PHBear You can join our new public channel on DCL chat. Will be using that for communications from now on.

**Comment by archtad** [comment]
Great update and look forward to the beginning of it all

**Comment by plato50** [comment]
Hi,

I Have contributed 10 land to the Fashion District.

1. How/when/where do I see my ownership you mentioned?
2. what channel do I need to go for information and discussion as I understand now there was a discussion with the contibutors?

Thx!

**Comment by submariner86** [comment]
Hey! As well participated with 10 land, excited !

**Comment by elpino** [comment]
Guys you should create a DIscord for this

**Comment by nkranders** [comment]
I would like to conribute as well but how do i do that

**Comment by bit802346** [comment]
nkranders i think later you can rent

**Comment by snickerr** [comment]
If you contributed in Fashion Street please join our Discord channel: https://discord.gg/xuDf6k7

**Comment by FazeelUsmani** [comment]
Hello, I wanted to contribute in Fashion street. How can I?

**Comment by Vladimir2341** [comment]
> Fashion Street - Project update 3
> 
> After listening to the contributors, we decided to give them ownership of part of the land contributed. We agree that owning some land in our district is probably the best investment right now before VR shopping takes off in a couple of years.
> 
> Part of the land contributed will be used by the district to build internal roads and plazas which will further increase the value of land. We will try to give land adjacent to a road to as much contributors as possible.
> 
> This means that there will not be any community owned buildings. In order to make profits from your store you need to work for it, have a dropshipping/affiliate contract with the brands and invest in marketing. It will work exactly like a Shopify store, but in in a decentralized VR world.
> 
> Everything else remains the same as stated in the initial description. If you want to have a turnkey store, our team will be happy to build it for you according to your needs. We have now different low poly store models to choose from. If you have development skills, you can choose a basic version of a store (think of it as a Shopify template) and decorate it with your own products or scan/create new items.
> 
> Fashion Street will maintain the decentralized spirit of DCL. So, if you can build stuff, you’re more than welcome to do that but it has to be in harmony with the district's theme. Unless your building is completely unrelated to the surrounding buildings, it will never be disapproved. If it’s unrelated to the district, we will submit it for community voting and it will be up to the community to make a decision. Remember that we aim to be “the elegant” district, so it’s probably going to be hard to pass the community test.
> 
> @PHBear You can join our new public channel on DCL chat. Will be using that for communications from now on.

hi? ploject is life?
